src/main/java/com/epam/rd/autotasks/springstatefulcalc/controller/WebCalculatorController.java

```diff
@@ -0,0 +1,88 @@
+package com.epam.rd.autotasks.springstatefulcalc.controller;
+
+import com.epam.rd.autotasks.springstatefulcalc.analyzer.analyzerFactory.LexemeAnalyzerFactoryImpl;
+import com.epam.rd.autotasks.springstatefulcalc.calculator.calculatorFactory.WebCalculatorFactoryImpl;
+import com.epam.rd.autotasks.springstatefulcalc.exception.BadRequestException;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Map;
+
+import static com.epam.rd.autotasks.springstatefulcalc.cacheData.CacheData.SINGLETON_DATA;
+import static com.epam.rd.autotasks.springstatefulcalc.constant.Constants.*;
+import static com.epam.rd.autotasks.springstatefulcalc.service.AppControllerService.APP_CONTROLLER_SERVICE;
+
+@Controller
+public class WebCalculatorController {
+
+    @PutMapping(MULTI_CALC_PATH)
```

Comment:
if you extract constants to the endpoint - have a separate constants class for the endpoints